[docs] Release Notes for 2.0.0a10 (#562)
someengineering/resoto: release-notes.md

# Release Notes 2.0.0a10 (Jan 18th, 2022)

> This release is exciting since it is the first one with a new name: Resoto.

Reviewer: ```suggestion
This release is exciting since it is the first one featuring our new product name: Resoto.
```

> Cloudkeeper is gone, and all `ck` names as well.

Reviewer: ```suggestion
Cloudkeeper is gone, along with all `ck` names.
```

> The components that make up Resoto are now called resotocore, resotoshell (resh), resotoworker, resotometrics.

Reviewer: ```suggestion
The components that make up Resoto are now called Resoto Core (`resotocore`), Resoto Shell (`resh`), Resoto Worker (`resotoworker`), and Resoto Metrics (`resotometrics`).
```

> Please note: also the docker image name has changed to [someengineering/resoto](https://github.com/someengineering/resoto/pkgs/container/resoto).

Reviewer: ```suggestion
*Note:* The Docker image name has also changed to [`someengineering/resoto`](https://github.com/someengineering/resoto/pkgs/container/resoto).
```
We should actually use the new Scarf image name now.  @lloesche?
